### PR TITLE
Acquia integration

### PR DESCRIPTION
**Pull request by Sanchiz**

### CODE REVIEW

File: files/acquia/jobs/ACQUIA_DEPLOY_BRANCH/config.xml.j2 [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  </scm>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers/>
+  <concurrentBuild>false</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command>cd new_pull_request
```
Comment (podarok) [inline files/acquia/jobs/ACQUIA_DEPLOY_BRANCH/config.xml.j2]: Newline, please

File: files/acquia/jobs/ACQUIA_RELEASE/config.xml.j2 [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        <reference></reference>
+        <timeout>5</timeout>
+      </hudson.plugins.git.extensions.impl.CloneOption>
+      <hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
+        <relativeTargetDir>new_pull_request</relativeTargetDir>
+      </hudson.plugins.git.extensions.impl.RelativeTargetDirectory>
+    </extensions>
+  </scm>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
```
Comment (podarok) [inline files/acquia/jobs/ACQUIA_RELEASE/config.xml.j2]: Let's keep disabled all jobs by default.
Except of PR builder and server cleaner

File: files/acquia/jobs/ACQUIA_SYNC_DB/config.xml.j2 [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties/>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
```
Comment (podarok) [inline files/acquia/jobs/ACQUIA_SYNC_DB/config.xml.j2]: true, please

File: files/acquia/jobs/ACQUIA_SYNC_FILES/config.xml.j2 [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties/>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
```
Comment (podarok) [inline files/acquia/jobs/ACQUIA_SYNC_FILES/config.xml.j2]: true, please

File: files/acquia/jobs/ACQUIA_UPDATE/config.xml.j2 [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties/>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
```
Comment (podarok) [inline files/acquia/jobs/ACQUIA_UPDATE/config.xml.j2]: true, please

File: files/acquia/jobs/NIGHTLY_DATABASE_DUMP/config.xml.j2 [hunk 6]
```diff
@@ -0,0 +1,53 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties/>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
```
Comment (podarok) [inline files/acquia/jobs/NIGHTLY_DATABASE_DUMP/config.xml.j2]: true, please

File: acquia.yml [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+      tags:
+        - update-jobs
+
+    - name: Copy UPDATE jobs
+      template: src=files/acquia/jobs/ACQUIA_UPDATE/config.xml.j2 dest=/var/lib/jenkins/jobs/ACQUIA_UPDATE_{{ item.environment_name }}/config.xml
+      with_items: servers
+      notify:
+        - 'Restart Jenkins'
+      tags:
+        - update-jobs
```
Comment (podarok) [inline acquia.yml]: Suffix   -jobs  sounds like redundant. Let's get rid of it.

File: acquia.yml [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+    # @todo create documentation about GitHub HOOKS for PR builder and GitHub plugins.
+    # @todo Drush rr before update
+    # @todo clear varnish
+    root_user: 'root'
+    jenkins_user: 'jenkins'
+
+    # ssh [PROJECT_NAME].[dev/test/prod/ra]@[SERVERS_VAR].[ACQUIA_CLOUD_HOSTING]
+    project_name: 'CHANGE_ME'
+    acquia_git_url: 'CHANGE_ME'
+    github_git_url: 'CHANGE_ME'
```
Comment (podarok) [inline acquia.yml]: Let's put all variables to specific config file. MVC pattern, please

**Comment by podarok** [comment]
We need to get rid of all CHANGE_ME placeholders, somehow

File: files/acquia/jobs/ACQUIA_RELEASE/config.xml.j2 (hunk 2 above)
Comment (Sanchiz) [inline files/acquia/jobs/ACQUIA_RELEASE/config.xml.j2]: I'm not sure that we can disable all jobs. This playbook won't be executed during provisioning.
Also I'm not sure about CHANGE_ME placeholders. We can't configure jobs for Acquia without servers https://github.com/propeoplemd/cibox/pull/236/files#diff-50fb31c783a15abe5897771c843c7f57R25

**Comment by Sanchiz** [comment]
Here's configuration that should be added for each project:
https://propeople-us.atlassian.net/wiki/pages/viewpage.action?pageId=18448418#CI/JenkinsServer-Acquiadeploymentscriptconfiguration(forDevOps):

**Comment by podarok** [comment]
@Sanchiz a few issues to be fixed

**Comment by podarok** [comment]
@Sanchiz inmplement one time var_prompt with yml config creation

**Comment by podarok** [comment]
Let's create steps for approach and split the task to small ones.
We should create abstraction about the acquia deployment plans as an option .

**Comment by podarok** [comment]
continuing in https://github.com/propeoplemd/cibox/pull/370

**Pull request closed without merging**